client/src/TenderList.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect } from "react";
+
+const TendersList = () => {
+	const [tenders, setTenders] = useState([]);
+	const [loading, setLoading] = useState(true);
+	const [page, setPage] = useState(1);
+	const [hasMore, setHasMore] = useState(true);
+
+	const fetchTenders = async (pageNumber) => {
+		setLoading(true);
+		try {
+			const response = await fetch(`/api/tenders?page=${pageNumber}`);
+			const data = await response.json();
+			setTenders((prevTenders) => [...prevTenders, ...data.tenders]);
+			setHasMore(data.tenders.length === 25);
```

Comment:
If the page has 25 items, then this doesn't necessarily mean there are more items. It's better if the server explicitly tells the client if there are more items

Reply:
I have done that